public/css/styles.css

```diff
@@ -65,3 +87,106 @@ canvas {
 .submenu-option.selected {
   color: var(--theme-color-3);
 }
+
+#desktop {
+  background: linear-gradient(
+    #C09DF7,
+    var(--theme-color-5)
+  );
+  height: 100vh;
+  font-family: 'Trebuchet MS', sans-serif;
+}
+
+.desktop-button {
+  box-sizing: border-box;
+  border: none;
+  background: var(--surface);
+  box-shadow: var(--border-raised-outer), var(--border-raised-inner);
+  min-width: 260px;
+  min-height: 30px;
+  margin-left: 5px;
+  font-family: 'Trebuchet MS', sans-serif;
+  font-size: 10pt;
+}
+
+.desktop-button:active {
+  box-sizing: border-box;
+  outline: 1pt;
+  outline-style: dotted;
+  outline-offset: -5px;
+  background: var(--surface);
+  box-shadow: var(--border-sunken-outer), var(--border-sunken-inner);
+  min-width: 260px;
+  min-height: 30px;
+  margin-left: 5px;
+  font-family: 'Trebuchet MS', sans-serif;
+  font-size: 10pt;
+}
```

Comment:
`:active` is a [pseudo-class](https://www.w3schools.com/css/css_pseudo_classes.asp) which is a way to provide alternative/extra styles to the rule it's attached to (in this case `.desktop-button`) in the case of a particular "event". which means u only need to include styles that are different from the main rule, the rest is redundant. so for example `outline: 1pt` is different so u would keep that, but `box-sizing: border-box` is redundant so u would remove that.